### DISCUSSION TITLE
OMP_NUM_THREADS and MKL_NUM_THREADS while running MACE with LAMMPS

### DISCUSSION THREAD

**Discussion by ShubhangG** [ACEsuit/mace#508] [General]
Hello,
I am trying to optimize using OMP_NUM_THREADS and MKL_NUM_THREADS to make my LAMMPS runs go quicker.

I wanted some clarity on the example used in the documentation. You use 56 and 56 for both these numbers, why? and why the same number?

**Comment by wcwitt** [comment]
Unfortunately, there isn't very much one can do right now (although we are working on it) to make MACE simulations fast on CPU. The OMP_NUM_THREADS and MKL_NUM_THREADS flags can improve threading performance for some operations, but the improvements are usually not dramatic. The example uses 56 because our machine has 56 cores.
